robot_skills/src/robot_skills/arms.py

```diff
@@ -646,7 +646,7 @@ def send_gripper_goal(self, state, timeout=5.0, max_torque=0.1):
             goal.command.direction = GripperCommand.OPEN
         elif state == GripperState.CLOSE:
             goal.command.direction = GripperCommand.CLOSE
-            goal.command.max_torque = max_torque
+            goal.command.max_force = max_force
```

Comment:
This is a msg, so change that too.

Reply:
it seems I indeed missed a few things.... not even only the msg.... Sorry for that! Good catch!